### PR TITLE
Concept entry luau string

### PR DESCRIPTION
**Pull request by cslylla**
### Description

- Added new concept to Luau: Strings
- Closes: #3210 

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/luau/concepts/strings/strings.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+## String Functions
+
+Luau provides various built-in string functions to manipulate and work with strings. The example below demonstrates the use of the `.upper()` string function to convert all characters in a string to uppercase:
+
+```lua
+local name = 'bilbo baggins'
+local capitalized = name.upper()
+
+print(capitalized)
```
Comment (caupolicandiaz) [inline content/luau/concepts/strings/strings.md:91]:
```suggestion
print(string.upper(name))
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@cslylla I've added my comments. Please update the entry, and tag me when you've finished making changes.

**Comment by cslylla** [comment]
@caupolicandiaz I updated the entry.

**Review by caupolicandiaz — APPROVED** [review]
@cslylla Looks good. This is ready for a second review

**Review by KTom101 — APPROVED** [review]
@cslylla Looks good to merge as is. Thanks for your patience.

**Pull request merged**